pkg/labels/labels.go

```diff
@@ -275,6 +275,7 @@ func (ls Labels) WithoutEmpty() Labels {
 		if v.Value != "" {
 			continue
 		}
+		// Copy slice without empty labels only if needed.
```

Comment:
I don't understand what this comment is trying to say.

Reply:
That we only touch the slice and copy things between if at least single label is empty (: Will rework comment

Reply:
reworded!